src/layouts/Layout.astro

```diff
@@ -61,6 +61,9 @@ const {
             fill="#fafafa"></path></svg
         >
       </a>
+      <div id="theme-toggle">
```

Comment:
We should replace this with series with group of checkboxes, dropdown menu or buttons with `aria-pressed` attribute. Unfortunately this approach is not accessible for assistive device users, nor for the screen readers and anything that relies on the markup.

Some examples of a well crafted markup for a theme switcher component:

- https://docs.astro.build/en/getting-started/
- https://pawelgrzybek.com

Reply:
Also, its position right next to the logo is a bit too prominent. Maybe we can move it level lower to the hero component, but this is not a big, styling we can discuss later, first we need to nail the functionality down.